docs/plugins.rst

```diff
@@ -78,6 +78,21 @@ looks like:
         menu_links = []
 
 
+You can derive it by inheritance (please refer to the example below). Please note ``name`` inside this class must be specified.
+
+After the plugin is imported into Airflow, you can invoke it using statement like
+
+.. code:: python
+
+    from airflow.{type, like "operators", "sensors", etc.}.{name specificed inside the plugin class} import *
+
+When you write your own plugins, make sure you understand them well. There are some essential properties for each type of plugin.
+For example,
```

Comment:
Perhaps missing an extra blank line above this?

Reply:
Hi @ashb , I have updated accordingly. You can check the final looking via https://github.com/XD-DENG/incubator-airflow/blob/patch-3/docs/plugins.rst#interface

Thanks.